modules/core/src/main/java/org/apache/fluo/core/observer/v1/ObserverStoreV1.java

```diff
@@ -209,7 +212,7 @@ public Observers getObservers(Environment env) {
       }
 
       @Override
-      public Set<Column> getObservedColumns(NotificationType nt) {
+      public ImmutableSet<Column> getObservedColumns(NotificationType nt) {
```

Comment:
Personally I would leave this type as `Set` and use `ImmutableSet` internally as the implementation.  However I can see that its useful to advertise that its immutable.